pkg/arm/data/master-startup.sh

```diff
@@ -25,6 +25,14 @@ EOF
   systemctl start docker.service
 fi
 
+# write the /root/.docker/config.json file to disk, but actually need it in
+# /var/lib/origin thanks to https://github.com/kubernetes/kubernetes/issues/45487
+DPATH=/root/.docker
+mkdir -p $DPATH
+base64 -d <<< {{ .Config.RootDockerConfig | Base64Encode }} >${DPATH}/config.json
```

Comment:
Is this something that we are going to be requiring everyone to use (even in development)? If not, you should set it optionally here.
```sh
{{- if ne .Config.RootDockerConfig "" }}
# write the /root/.docker/config.json file to disk, but actually need it in
# /var/lib/origin thanks to https://github.com/kubernetes/kubernetes/issues/45487
DPATH=/root/.docker
mkdir -p $DPATH
base64 -d <<< {{ .Config.RootDockerConfig | Base64Encode }} >${DPATH}/config.json
chmod 0600 ${DPATH}/config.json
ln -s ${DPATH} /var/lib/origin/.docker
{{ end }}
```
Otherwise, you should require it during validation?

Reply:
The plan was to share the file same as with the rest of the `secrets/` directory

Reply:
Can you make sure the release repo gets updated with the new secret requirement? You will need to update the Makefile, projects/azure/README.md, and https://github.com/openshift/release/blob/master/ci-operator/SECRETS.md#cluster-secrets-azure

Reply:
openshift/release#2584